components/brave_new_tab_ui/components/default/braveToday/customize/FollowButton.tsx

```diff
@@ -29,13 +28,11 @@ const StyledButton = styled.div<{ following: boolean }>`
 export default function FollowButton (props: Props) {
   const { following, className, ...rest } = props
   return <StyledButton className={className} following={following}>
-    <Button {...rest} isPrimary={!following}>
-      {following ? <>
-        {Heart} {getLocale('braveNewsFollowButtonFollowing')}
-      </>
-        : <>
-          {HeartOutline} {getLocale('braveNewsFollowButtonNotFollowing')}
-        </>}
+    <Button {...rest} scale='tiny' isPrimary={!following}>
```

Comment:
:+1: nice to see the button getting less loud

Reply:
We can replace this with Leo's button when we have that available and maybe use that opportunity to clean all this up a bit and reduce repetition